### ISSUE TITLE
Identify "best" results from heuristic thinning?

### ISSUE THREAD

**Issue by mlammens** [mlammens/spThin#12]
@paleo13 is there a way to easily identify which of the datasets resulting from the heuristic thinning match the greatest number of records returned?

**Comment by paleo13** [comment]
Good point. The summary method prints the index of the best solution, but this isn't great if you want to automate it.

I've changed the [[ method so that if the 0th index is accessed this return the best solution.

I've also updated the vignette to demonstrate this, since the help file for [[.SpPartial is a bit obscure.
